ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh

```diff
@@ -113,6 +113,22 @@ registry.redhat.io/rhel8/httpd-24:latest=MIRROR_REGISTRY_PLACEHOLDER/rhel8/httpd
 
 }
 
+function check_mirror_registry () {
+    run_command "oc adm new-project sample-test"
+    ret=0
+    run_command "oc import-image mytestimage --from=quay.io/openshifttest/busybox@sha256:c5439d7db88ab5423999530349d327b04279ad3161d7596d2126dfb5b02bfd1f --confirm -n sample-test" || ret=$?
```

Comment:
s/||/;/ ?

Reply:
I guess no need it.

Reply:
Why?
Per the above logic, only when oc command failed, ret will be assigned, if that, the following if block sounds like missing something.